lib/commons/color/get-background-color.js

```diff
@@ -38,16 +46,45 @@ function getBgColor(elm, elmStyle) {
 	return bgColor;
 }
 
-function calculateObscuringAlpha(elmIndex, elmStack) {
+/**
+ * Determines overlap of node's content with a bgNode. Used for inline elements
+ * @private
+ * @param {Element} targetElement
+ * @param {Element} bgNode
+ * @return {Boolean}
+ */
+function contentOverlapping(targetElement, bgNode) {
```

Comment:
The DOM order/z-index of the elements matters when you do the comparison, and I couldn't get that to work for all scenarios.